iguana/xml_writer.hpp

```diff
@@ -102,6 +96,21 @@ template <typename Stream> inline void render_head(Stream &ss, const char *s) {
   ss.push_back('>');
 }
 
+template <typename Stream, typename T>
+inline void render_xml_value0(Stream& ss, const T& v, std::string_view name) {
+  for (auto& item : v) {
+    using item_type = std::remove_cvref_t<decltype(item)>;
+    if constexpr (is_reflection_v<item_type>) {
+      to_xml_impl(ss, item, name);
+    }
+    else {
+      render_head(ss, name.data());
+      render_xml_value(ss, item);
+      render_tail(ss, name.data());
```

Comment:
such code is duplicate code, please refactor the code(put the code in a method: render_xml_node).